### PR TITLE
Fix #377: Control submit button

### LINKED ISSUE #377: Submit and continue buttons need proper disabled colors
It's not currently clear in the exploration player when an answer can't be submitted due to us not having distinct disabled colors.

We should use color selectors for this purpose.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR controls the styling of "Submit " button.
1. If answer is not entered -> Submit button is inactive. (Though this UI of inactive submit button does not match with [mocks](https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/4744bfae-84be-44fa-a27d-9a252fdfc2f9/PM-Q1-Fraction-Input-No-Answer-), but this was discussed with Chantel to make this similar to `Start` button of `TopicTrainTab`)
Screenshot 2019-11-19 at 6 12 45 PM

2. If answer is entered -> Submit button is active.
Screenshot 2019-11-19 at 6 12 59 PM

Also, I have found one issue while working on this PR which has been mentioned in #417 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by nikitamarysolomanpvt — COMMENTED** [review]
small
While the input is empty submit button should be active color?

**Review by nikitamarysolomanpvt — COMMENTED** [review]
PTAL on the comment

**Comment by BenHenning** [comment]
Could we address this via data binding and the top level view model, instead? See https://github.com/oppia/oppia-android/blob/develop/app/src/main/java/org/oppia/app/player/state/StateViewModel.kt#L33 

We ought to not have classes directly depend their parents. We could use listeners to indirect this, but there may be a cleaner reactive solution possible.

File: app/src/main/java/org/oppia/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt [hunk 1]
```diff
@@ -6,13 +6,14 @@ import org.oppia.app.model.Interaction
...
 import org.oppia.app.player.state.SelectionItemInputType
+import org.oppia.app.player.state.StateFragmentPresenter
 import org.oppia.app.player.state.answerhandling.InteractionAnswerHandler
 import org.oppia.app.player.state.answerhandling.InteractionAnswerReceiver
 
 /** ViewModel for multiple or item-selection input choice list. */
 class SelectionInteractionViewModel(
   val explorationId: String, interaction: Interaction, private val interactionAnswerReceiver: InteractionAnswerReceiver,
-  existingAnswer: InteractionObject?, val isReadOnly: Boolean
+  existingAnswer: InteractionObject?, val isReadOnly: Boolean, @Suppress("UNUSED_PARAMETER") stateFragmentPresenter: StateFragmentPresenter
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt:16]: use cmd+option+L to wrap this.

**Review by veena14cs — CHANGES_REQUESTED** [review]
Please update this code with latest develop as there are too many changes.

**Comment by rt4914** [comment]
Closing this PR as this will be done once `temp-integration-pt-3` code is in develop.

**Pull request closed without merging**